app/assets/stylesheets/fonts.scss

```diff
@@ -0,0 +1,27 @@
+/* roboto-regular - latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: url('roboto-v29-latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('roboto-v29-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
```

Comment:
Kinda curious as to whether we need shims that go back this far. (Of course, now that I've said it, we'll see thousands of IE6 user agents in the logs.)